src/iop/retouch.c

```diff
@@ -1698,6 +1787,21 @@ static gboolean rt_select_algorithm_callback(GtkToggleButton *togglebutton, GdkE
 
   dt_dev_add_history_item(darktable.develop, self, TRUE);
 
+  // if we have the shift key pressed, we set it as default
+  if(dt_modifier_is(e->state, GDK_SHIFT_MASK))
+  {
+    dt_conf_set_int("plugins/darkroom/retouch/default_algo", p->algorithm);
+    // and we show a toat msg to confirm
+    if(p->algorithm == DT_IOP_RETOUCH_CLONE)
+      dt_toast_log(N_("default tool changed to %s"), N_("cloning"));
```

Comment:
This should be dt_control_log and not dt_toast_log. Same for all occurrences. The toast log is for very ephemeral information like changing opacity and is display top-center. The control log is where such information should be sent to be consistent.

Reply:
On reflection I agree. If your attention is on the right-hand panel (clicking buttons), dt_toast_log can easily be missed.

Reply:
agree... and done :)